develop/tutorials/articles/110-portlets/15-javascript/04-npm/02-adding-liferay-npm-bundler-to-your-portlet.markdown

```diff
@@ -0,0 +1,63 @@
+# Adding liferay-npm-bundler to Your Portlet [](id=adding-liferay-npm-bundler-to-your-portlet)
+
+Adding liferay-npm-bundler to your portlet involves installing the package, via 
+npm, and adding it to your npm build process. Get started by installing 
+liferay-npm-bundler next.
+
+## Installing liferay-npm-bundler [](id=installing-liferay-npm-bundler)
+
+Follow these steps to install liferay-npm-bundler:
+
+1.  Install NodeJS >= [v6.11.0](http://nodejs.org/dist/v6.11.0/) if you 
+    don't have it installed.
+
+2.  Navigate to your portlet's `src/main/resources/META-INF/resources` 
+    folder.
+     
+    If you don't have a portlet already, create an empty MVC portlet. For 
+    convenience, you can use [Blade CLI](https://dev.liferay.com/develop/tutorials/-/knowledge_base/7-0/installing-blade-cli) 
+    to create an empty portlet with the [mvc portlet blade template](https://dev.liferay.com/develop/reference/-/knowledge_base/7-0/using-the-mvc-portlet-template).
+
+3.  Run the following command to install the liferay-npm-bundler:
+
+        npm install --save-dev liferay-npm-bundler
+
+    +$$$
+
+    **Note:** Use npm from within your portlet's 
```

Comment:
This is "incorrect": you don't need to use npm from 'src/main/resources/.....'. You use it from your project's root folder, where the package.json file lives.